source/install-using-precompiled-files/index.html.md.erb

```diff
@@ -10,26 +10,26 @@ You can install GOV.UK Frontend by copying our CSS, JavaScript and asset files i
 <%= warning_text('In your live application, you should <a href="/installing-with-npm/">install with Node.js package manager (npm)</a> instead.
 ') %>
 
-## What you cannot do
+## What you cannot do after installing
 
-You will not be able to:
+You’ll not be able to:
 
-- change Sass settings, for example you will not be able to override colours or set your own font
-- use the Nunjucks macros from the [Design System website](https://design-system.service.gov.uk/)
+- change [Sass settings](/sass-api-reference/), for example override colours or set your own font
+- use the Nunjucks code from the [Design System website](https://design-system.service.gov.uk/) to add components
 - import an individual component’s CSS or JavaScript
 - use GOV.UK Frontend's colours or mixins in your custom code
 
 ## Copy the files
 
 1. Download the `release-<VERSION-NUMBER>.zip` file at the bottom of the [latest GOV.UK Frontend release note](https://github.com/alphagov/govuk-frontend/releases/latest).
 2. Unzip the zip file.
-3. Copy the `assets` folder to the root of your project’s public folder, so that for example `<YOUR-SITE-URL>/assets/govuk-logotype-crown.png` will work in your users’ browsers.
-4. Copy the 2 `.css` files to a stylesheets folder in the root of your project’s public folder, so that for example `<YOUR-SITE-URL>/stylesheets/govuk-frontend-<VERSION-NUMBER>.min.css` will work in your users’ browsers.
-5. Copy the `.js` file to a JavaScript folder in the root of your project’s public folder, so that for example `<YOUR-SITE-URL>/javascript/govuk-frontend-<VERSION-NUMBER>.min.js` will work in your users’ browsers.
+3. Copy the `assets` folder to the root of your project’s public folder, so that for example `<YOUR-SITE-URL>/assets/govuk-logotype-crown.png` shows the `govuk-logotype-crown.png` image in your users’ browsers.
```

Comment:
Factually correct, so feel free to ignore this, but could it just be ‘a browser’?

I think talking about the user’s browser is confusing… it’s not like you’re going to ask a user to check, and the user would never actually go and look at the CSS or JS.